src/templates/expressions/fields.test.ts

```diff
@@ -160,22 +118,10 @@ describe("SubTableField", () => {
 describe("FileField", () => {
     test("toTsExpression()", () => {
         expect(
-            new FileField("fieldName", "FILE")
-                .tsExpression()
-                .trim()
+            new FileField("fieldName").tsExpression().trim()
         ).toEqual(
             `
-"fieldName" : {
-    type: "FILE";
-    value: {
-        contentType: string;
-        fileKey: string;
-        name: string;
-        size: string;
-    }[];
-    disabled?: boolean;
-    error?: string;
-};`.trim()
+"fieldName" : kintone.types.fields.File;`.trim()
```

Comment:
It looks like you deleted unnecessary line break in 0ec46dc , so what about deleting this, too?